tests/new_unit_tests/Commands/Env/MockCloneCommand.php

```diff
@@ -0,0 +1,8 @@
+<?php
+namespace Pantheon\Terminus\UnitTests\Commands\Env;
+
+use Pantheon\Terminus\Commands\CloneCommand;
+
+class MockCloneCommand extends CloneCommand
```

Comment:
This is simply a test class so that I can test the protected methods of the abstract class